pglookout/pglookout.py

```diff
@@ -154,6 +154,13 @@ def load_config(self, _signal=None, _frame=None):
         self.replication_lag_failover_timeout = self.config.get("max_failover_replication_time_lag", 120.0)
         self.replication_catchup_timeout = self.config.get("replication_catchup_timeout", 300.0)
         self.missing_master_from_config_timeout = self.config.get("missing_master_from_config_timeout", 15.0)
+
+        if self.replication_lag_warning_boundary >= self.replication_lag_failover_timeout:
+            msg = "Replication lag warning boundary (%s) is not lower than its failover timeout (%s)"
+            self.log.warning(msg, self.replication_lag_warning_boundary, self.replication_lag_failover_timeout)
+            self.replication_lag_warning_boundary = self.replication_lag_failover_timeout
```

Comment:
A bit on the nitpicky side but I'd only do this and the logging below if warning boundary is greater than failover boundary. The logging below is confusing when the values are the same as nothing was changed. The logging above is still sensible if the limits are the same.

Reply:
agreed